.travis.yml

```diff
@@ -1,9 +1,13 @@
 languages: java
-# TODO: Add openjdk8 once it is available in Travis.
 jdk:
-  - oraclejdk8
-dist: trusty
-sudo: false
+  - openjdk8
+os: linux
+dist: xenial
+
+before_install:
+  - sudo apt update
+  - sudo apt install openjfx
+
```

Comment:
oraclejdk8 isn't supported on Xenial (https://travis-ci.community/t/install-of-oracle-jdk8-is-failing/4365)
OpenJDK 8 doesn't include JavaFX (https://stackoverflow.com/questions/35974003/javafx-comes-with-jdk-8)